src/Apps/Order/Components/AddressVerificationFlow.tsx

```diff
@@ -392,28 +462,23 @@ export const AddressVerificationFlowQueryRenderer: React.FC<{
   return (
     <SystemQueryRenderer<AddressVerificationFlowQuery>
       variables={{ address }}
-      render={({ props, error }) => {
-        if (error) {
-          // TODO: we need better error handling
-          console.error(error)
-          return null
-        }
-
```

Comment:
I'm not sure about merging this - I haven't seen an error manifest here in testing (which confused me) but if it did happen the entire address submission would be blocked. Could we return a fake unavailable value as a fallback in this case?

Reply:
We could mock `props` to look like a `REVIEW_AND_CONFIRM` situation...

Reply:
Yes, I'm going to ticket this thread for followup since I don't even know how to reproduce it in staging for testing.